README.md

```diff
@@ -39,6 +39,12 @@ Run this command in the root of project directory to install NimblePhxGenTemplat
 mix do deps.get, deps.compile
 ```
 
+Step 4: Run migrate
+
+```bash
+mix ecto.migrate
```

Comment:
@byhbt I think the ecto.migrate isn't a part of this template, also the section there is for install this template, not relate to ecto topic.

In the future we also support non-ecto projects so this readme is not needed IMO

Reply:
hi @andyduong1920 !

Yes, I removed it, thanks for the info.